app/src/main/res/layout/onboarding_whats_new.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<androidx.constraintlayout.widget.ConstraintLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:id="@+id/onboarding_card"
+        style="@style/OnboardingCardLight"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+    <TextView
+            android:id="@+id/header_text"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:text="@string/onboarding_whats_new_header1"
+            tools:drawableStart="@drawable/ic_whats_new"
+            android:drawablePadding="12dp"
+            android:textAppearance="@style/HeaderTextStyle"
+            android:gravity="center_vertical"
+            android:lines="1"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintEnd_toEndOf="parent" />
+
+    <TextView
+            android:id="@+id/description_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:textAppearance="@style/Body14TextStyle"
+            android:layout_marginTop="12dp"
+            android:text="@string/onboarding_whats_new_description"
+            app:layout_constraintTop_toBottomOf="@id/header_text"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintBottom_toTopOf="@id/get_answers"/>
+    <TextView
+            android:id="@+id/get_answers"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:textAppearance="@style/Body14TextStyle"
+            android:paddingTop="12dp"
+            android:paddingBottom="12dp"
+            android:text="@string/onboarding_whats_new_description_linktext"
```

Comment:
ditto
```suggestion
            tools:text="@string/onboarding_whats_new_description_linktext"
```

Reply:
Here an underline span is applied, but the text is taken from the textview, so we will still need android:text.